app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```
@@ -2,104 +2,117 @@
   background: govuk-colour("white");
   margin-top: govuk-spacing(6);
 
-  @include govuk-media-query($from: tablet) {
+  @include govuk-media-query($from: desktop) {
     margin-top: govuk-spacing(9);
-    border-bottom: 1px solid govuk-colour("white");
   }
 
+  // New design has the box flush with edges of smaller screens
+  // We need to compensate for `govuk-width-container` margins:
+  @include govuk-media-query($until: tablet) {
+    margin-right: -15px;
+    margin-left: -15px;
```

Could these use the spacing helper?

```suggestion
    margin-right: govuk-spacing(-3);
    margin-left: govuk-spacing(-3);
```

---

Apologies, I didn't notice the 'do not review' label - ignore me!

---

Good point, thanks. It's possible these margins might change, based on design feedback. I'll wait to apply your suggestion just in case it's not needed at all.